Sources/Bar.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Bar.swift
+//  Pods
+//
+//  Created by Stijn Willems on 31/08/16.
+//
+//
+
+import Foundation
+
+
+/// Serves anything you order.
+
+public class Bar <S: Serveable>  {
+
+    public let configuration: Configuration
+    public let service : S
+
+    public init (configuration: Configuration, service: S) {
```

Comment:
Because now you always have to set the service. I would prefer to have a default one, that can be changed if needed. But the default should work.

Check my comment about `Serveable`, I thought we were going to use it in a different way...